Vodovoz/Core/Permissions/SubdivisionForUserEntityPermissionWidget.cs

```diff
@@ -196,5 +222,91 @@ public void Save()
 				uow.Delete(item);
 			}
 		}
+
+		#region Search
+
+		public void SearchSubdivisions(string searchString, RepresentationTreeView treeToSearch)
+		{
+			if(searchString != "")
+			{
+				var items = treeToSearch.RepresentationModel;
```

Comment:
а не проще явно задать IList<SubdivisionVMNode> тут?

Reply:
Ну вообще да, красивей будет как мне кажется сделать 
```suggestion
				var items = treeToSearch.RepresentationModel.ItemsList as IList<SubdivisionVMNode>;
```

Reply:
Изменил